File: app/src/main/java/com/example/gestionturnosapp/data/Medicamento.kt
package com.example.gestionturnosapp.data

data class Medicamento(
    val id: String,
    val nombre: String,
    val dosis: String,
    val frecuencia: String, // ej: "Cada 8 horas"
    val proximaToma: String,
    val notas: String? = null
)

File: backend/models/Medicamento.js
const mongoose = require('mongoose');

const MedicamentoSchema = new mongoose.Schema({
    usuarioId: { type: mongoose.Schema.Types.ObjectId, ref: 'User', required: true },
    nombre: { type: String, required: true },
    dosis: { type: String, required: true },
    frecuencia: { type: String, required: true },
    proximaToma: { type: String, required: true },
    notas: { type: String, default: "" }
});

module.exports = mongoose.model('Medicamento', MedicamentoSchema);
